assets/dbs/npc_db.json

```diff
@@ -464,5 +464,28 @@
                 "loop": [true,true]
             }
         }
+    },{
+        "key_name": "psynergy_stone",
+        "anchor_y": 0.5,
+        "initial_action": "stone",
+        "collision_body_bevel": 3,
+        "interaction_pattern": "tik_tak_toe",
+        "no_shadow": true,
+        "body_radius": 7.0,
+        "scale_x": 1.0,
+        "scale_y": 1.0,
+        "actions": {
+            "stone": {
+                "frame_rate": [[0.5,10,10,10,10,10,10],6],
```

Comment:
The shining animation is a little bit faster. You can try setting it to 20 instead of 10 and seems whether it's closer to the original.

Reply:
The `"shatter"` animation is also a little bit faster.